examples/migration/recursive_fibonacci/task_emulation_layer.h

```diff
@@ -0,0 +1,145 @@
+/*
+    Copyright (c) 2023 Intel Corporation
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+#ifndef __TBB_task_emulation_layer_H
+#define __TBB_task_emulation_layer_H
+
+#include "tbb/task_group.h"
+#include "tbb/task_arena.h"
+
+#include <atomic>
+
+namespace task {
```

Comment:
To make it clearer to the reader of the example what is a native part of oneTBB vs emulated, perhaps use a different namespace like task_emulation. Then at use point(s), use the full namespace. This will help distinguish between uses of fully supported features, such as task_group, from the emulation layer.

Reply:
Make sense. Changed namespace name on `task_emulation`